_posts/2019-01-29-hello-world.md

```diff
@@ -1,8 +1,75 @@
 ---
 title: "Welcome to Jekyll!"
-published: false
+published: true
```

Comment:
Did you mean to change published to true here?

Reply:
In fact, I am not very confident with this change. 
At first i thought this was a good idea to publish a first post with examples of highlighter. Besides this modification can enable an easier review of this PR. But some users may not want a first post, i don't know for the moment.
If you suggest to revert the published state to "false", i wouldn't mind doing it.